webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
```

Comment:
I've also heard the claim before that recursing ends up including everything, but this does not seem to be true from what I've found, because links tend to point "down" into the stack, not up. As long as we preclude stats for *other* tracks, I think there are ways to implement this that will be performant. The only real variable here is the number of tracks in a connection, which is why the track selector works imho.